### PR TITLE
Stylecop

### PR DESCRIPTION
**Pull request by bokobza**
- added this. where relevant
- added missing access modifiers
- sorted using statements
- fixed line spacing
- renamed constants to use pascal casing
- use auto-property where possible
- remove unnecessary type declaration

### CODE REVIEW

File: src/Stratis.Bitcoin.Features.Consensus/ConsensusPerformanceCounter.cs [hunk 1]
```diff
@@ -10,15 +10,16 @@ public class ConsensusPerformanceSnapshot
     {
         public ConsensusPerformanceSnapshot(long processedInputs, long processedTransactions, long processedBlocks, long blockFetchingTime, long blockProcessingTime, long utxoFetchingTime)
         {
-            this._TotalProcessedTransactions = processedTransactions;
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.Consensus/ConsensusPerformanceCounter.cs]: Shouldn't we rename these as well?

File: src/Stratis.Bitcoin.Features.Consensus/ConsensusPerformanceCounter.cs [hunk 2]
```diff
@@ -10,15 +10,16 @@ public class ConsensusPerformanceSnapshot
...
+            this.TotalProcessedTransactions = processedTransactions;
             this._TotalProcessedInputs = processedInputs;
-            this._TotalProcessedBlocks = processedBlocks;
+            this.TotalProcessedBlocks = processedBlocks;
             this._TotalBlockFetchingTime = blockFetchingTime;
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.Consensus/ConsensusPerformanceCounter.cs]: Same here etc...

File: src/Stratis.Bitcoin.Features.Consensus/ConsensusPerformanceCounter.cs [hunk 3]
```diff
@@ -203,7 +187,8 @@ public ConsensusPerformanceCounter(IDateTimeProvider dateTimeProvider)
             this._Start = this.dateTimeProvider.GetUtcNow();
         }
 
-        long _ProcessedTransactions;
+        internal long _ProcessedTransactions;
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.Consensus/ConsensusPerformanceCounter.cs]: Same here.

File: src/Stratis.Bitcoin.Features.Consensus/ConsensusPerformanceCounter.cs [hunk 4]
```diff
@@ -293,14 +287,16 @@ public ConsensusPerformanceSnapshot Snapshot()
             return snap;
         }
 
-        DateTime _Start;
+        internal DateTime _Start;
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.Consensus/ConsensusPerformanceCounter.cs]: Loads in this file class ;)

File: src/Stratis.Bitcoin.Features.LightWallet/Settings.StyleCop [hunk 5]
```diff
@@ -1,796 +0,0 @@
-<StyleCopSettings Version="105">
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.LightWallet/Settings.StyleCop]: So how does work then? Where did the stylecop settings for this move to?

(Sorry the diff is a bit big :P)

File: src/Stratis.Bitcoin.Features.MemoryPool/TxMemPoolEntry.cs [hunk 6]
```diff
@@ -30,7 +30,9 @@ public class LockPoints // todo: replace with SequenceLock
         /// values are still valid even after a reorg.
         /// </remarks>
         public ChainedBlock MaxInputBlock;
-    };
+    }
+
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.MemoryPool/TxMemPoolEntry.cs]: Maybe one too many extra empty lines here?

File: src/Stratis.Bitcoin.Features.Miner/PowBlockAssembler.cs [hunk 7]
```diff
@@ -42,7 +50,9 @@ public BlockTemplate()
             this.VTxFees = new List<Money>();
             this.TxSigOpsCost = new List<long>();
         }
-    };
+    }
+
+;
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.Miner/PowBlockAssembler.cs]: Extra line

File: src/Stratis.Bitcoin.Features.Miner/PowBlockAssembler.cs [hunk 8]
```diff
@@ -59,10 +69,15 @@ public TxMemPoolModifiedEntry(TxMempoolEntry entry)
...
             public TxMempoolEntry iter;
+
             public long SizeWithAncestors;
+
             public Money ModFeesWithAncestors;
+
             public long SigOpCostWithAncestors;
-        };
+        }
+
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.Miner/PowBlockAssembler.cs]: Extra line

File: src/Stratis.Bitcoin/Utilities/UnspentOutputs.cs [hunk 9]
```diff
@@ -8,19 +8,19 @@ public class UnspentOutputs : IBitcoinSerializable
...
         public UnspentOutputs()
         {
-
         }
+
         public UnspentOutputs(uint height, Transaction tx)
         {
             Guard.NotNull(tx, nameof(tx));
 
             this._Outputs = tx.Outputs.ToArray();
```
Comment (fassadlr) [inline src/Stratis.Bitcoin/Utilities/UnspentOutputs.cs]: How about renaming this as well?

File: src/Stratis.Bitcoin/Utilities/UnspentOutputs.cs [hunk 10]
```diff
@@ -8,19 +8,19 @@ public class UnspentOutputs : IBitcoinSerializable
...
             this._TransactionId = tx.GetHash();
```
Comment (fassadlr) [inline src/Stratis.Bitcoin/Utilities/UnspentOutputs.cs]: And this.

**Comment by bokobza** [comment]
@fassadlr oh yes, it's not finished by a long way.
This is just the first shot, more is coming.
I've taken care of these comments and removed all the underscores from fields and parameters, as well as the extra lines you noticed. For some reason, these didn't get picked up because of the ; that was lost in the middle of the file.

**Comment by bokobza** [comment]
And the stylecop file has moved to the solution level.

**Pull request merged**